### PR TITLE
chore: refactor docs

### PR DESCRIPTION
**Pull request by lucasjs**

### CODE REVIEW

File: src/Button/index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import ThemeConsumer from '../ThemeConsumer'
 const consumeTheme = ThemeConsumer('UIButton')
 
 /**
- * Simple html button in a beautifull skin.
+ * Simple HTML button in a beautifull skin.
```
Comment (derekstavis) [inline src/Button/index.js]: `beautiful` só tem um L :stuck_out_tongue:

File: src/DateInput/index.js [hunk 2]
```diff
@@ -54,11 +54,11 @@ const getStrings = strings => ({
...
 /**
  * Input component designed to receive one or two dates, can have a mask
- * for the input and enables the date selection using a calendar like selector
- * When dates are selected the first one is the period start and the second is
- * the finish, the date selection callback allways will provide two dates, if
- * the component is showing only one date the callback will receive the same date
- * as start and end
+ * for the input and enables the date selection using a calendar like selector.
+ * When dates are selected, the first one is the period start and the second is
+ * the finish. The date selection callback allways will provide two dates, if
```
Comment (derekstavis) [inline src/DateInput/index.js]: `s/finish/end` e `s/allways/always`

Comment (derekstavis) [reply src/DateInput/index.js]: `... will always provide ...`

File: src/DateInput/index.js [hunk 3]
```diff
@@ -54,11 +54,11 @@ const getStrings = strings => ({
...
+ * the component is showing only one date the, callback will receive the same date
```
Comment (derekstavis) [inline src/DateInput/index.js]: dangling `,`

File: src/SegmentedSwitch/index.js [hunk 4]
```diff
@@ -86,7 +86,7 @@ SegmentedSwitch.propTypes = {
   items: arrayOf(string).isRequired,
   /**
    * The prop responsible for identifying the selected item.
-   * Its value is the value of one of the "items".
+   * Its value is the value of one of the 'items'.
```
Comment (derekstavis) [inline src/SegmentedSwitch/index.js]:
```
`items`
```

File: src/Button/index.js (hunk 1 above)
Comment (lucasjs) [reply src/Button/index.js]: Done.

File: src/DateInput/index.js (hunk 3 above)
Comment (lucasjs) [reply src/DateInput/index.js]: Done.

**Pull request merged**